### ISSUE TITLE
`houndd -version` returns v0.4.0 on v0.5.0

### ISSUE THREAD

**Issue by ictrobot** [hound-search/hound#425]
```
> go install github.com/hound-search/hound/cmds/houndd@v0.5.0
go: downloading github.com/hound-search/hound v0.5.0
go: downloading github.com/blang/semver v3.5.1+incompatible

> houndd -version
houndd v0.4.0
```
Presumably it should say `houndd v0.5.0`? It appears the version number is just hardcoded and was not updated

https://github.com/hound-search/hound/blob/27fa1efd2fad3216acc41da5ed7998cff1c2744c/cmds/houndd/main.go#L119-L123

**Comment by salemhilal** [comment]
Ah yes, this is something we always forget to do -_- 
Thank you for noticing! I'll update and release a patch.

**Comment by salemhilal** [comment]
This should be fixed as of ae638e5b4793bae392202e645a931ea2044ea26d. v0.5.1 is building as we speak. You can follow along with the release [here](https://github.com/hound-search/hound/actions/runs/2542437380), if you'd like. Thanks for reporting!
